charts/redhat/redhat/redhat-dotnet-imagestreams/0.0.3/src/templates/tests/test-dotnet.yaml

```diff
@@ -16,7 +16,14 @@ spec:
       command:
         - '/bin/bash'
         - '-ec'
-        - 'dotnet --list-runtimes | grep -F {{ $latestversion }}. && [[ -z $(dotnet --list-sdks) ]]'
+        - 'dotnet --list-runtimes | grep -F {{ $latestversion }}. && ! dotnet --list-runtimes | grep -q Microsoft.AspNetCore.App && [[ -z $(dotnet --list-sdks) ]]'
```

Comment:
This check was updated for latest runtime to explicitly check there is no ASP.NET Core runtime.